src/config/config.ts

```diff
@@ -2,6 +2,20 @@ export const DOCUMENT_STORE_CREATOR_ROPSTEN = "0x4077534e82C97Be03A07FB10f5c853d
 export const DOCUMENT_STORE_CREATOR_MAINNET = "0x0";
 export const PROXY_FACTORY_ROPSTEN = "0xba2501bf20593f156879c17d38b6c245ca65de80";
 export const PROXY_FACTORY_MAINNET = "0x0";
+export const TITLE_ESCROW_CREATOR_ROPSTEN =
+  process.env.TITLE_ESCROW_CREATOR_ROPSTEN || "0xB0dE5E22bAc12820b6dbF6f63287B1ec44026c83";
+export const TITLE_ESCROW_CREATOR_MAINET =
+  process.env.TITLE_ESCROW_CREATOR_MAINET || "0x907A4D491A09D59Bcb5dC38eeb9d121ac47237F1";
+export const TITLE_ESCROW_CREATOR_RINKEBY =
+  process.env.TITLE_ESCROW_CREATOR_RINKEBY || "0xa51B8dAC076d5aC80507041146AC769542aAe195";
+export const TITLE_ESCROW_CREATOR_LOCAL =
```

Comment:
leave process.env out, this should describe the deployed one and not be ambiguous about it